osu.Game/Skinning/Editor/SkinBlueprint.cs

```diff
@@ -2,56 +2,105 @@
 // See the LICENCE file in the repository root for full licence text.
 
 using osu.Framework.Allocation;
+using osu.Framework.Extensions.Color4Extensions;
 using osu.Framework.Graphics;
 using osu.Framework.Graphics.Containers;
 using osu.Framework.Graphics.Primitives;
 using osu.Framework.Graphics.Shapes;
 using osu.Game.Graphics;
-using osu.Game.Graphics.UserInterface;
+using osu.Game.Graphics.Sprites;
 using osu.Game.Rulesets.Edit;
 using osuTK;
+using osuTK.Graphics;
 
 namespace osu.Game.Skinning.Editor
 {
     public class SkinBlueprint : SelectionBlueprint<ISkinnableComponent>
     {
         private Container box;
 
+        private Container outlineBox;
+
         private Drawable drawable => (Drawable)Item;
 
         /// <summary>
         /// Whether the blueprint should be shown even when the <see cref="SelectionBlueprint{T}.Item"/> is not alive.
         /// </summary>
-        protected virtual bool AlwaysShowWhenSelected => false;
+        protected virtual bool AlwaysShowWhenSelected => true;
+
+        protected override bool ShouldBeAlive => drawable.IsAlive && Item.IsPresent;
```

Comment:
What's happened here? `AlwaysShowWhenSelected`'s value is changed here, but the property is also not read any longer after this change? Seems like either the virtual property should be removed, or the `ShouldBeAlive` change should be reverted, unless this is somehow needed for further changes downstream?

Reply:
Didn't realise it wasn't an override. Looks like accidental copy paste from the beatmap editor implementation (and can be removed).